dashboard/ios/Runner/GoogleService-Info.plist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>API_KEY</key>
+	<string>AIzaSyAOGTpCjagNvzVxgfsxMmISMN0Wgk2Wb7o</string>
```

Comment:
Is this API key sensitive?

https://blog.gitguardian.com/leaking-secrets-on-github-what-to-do/